wiki/News_Styling_Criteria/en.md

```diff
@@ -1,13 +1,19 @@
-# News
+# News Styling Criteria
 
-News articles exist on a slightly different system to the main osu!wiki, but follow similar principles. Writing them involves careful attention to a few key features and formats as outlined below.
+*For wiki articles, see [Article Styling Criteria](/wiki/Article_Styling_Criteria)*
+
+News articles exist on a slightly different system to the main osu! wiki, but follow similar principles. Writing them involves careful attention to a few key features and formats as outlined below.
 
 As always, all news articles should aim to have proper spelling and grammar at a bare minimum, as well as having correct and up-to-date information.
 
 ## Repository directory
 
 The following standards apply only to articles written for the news, and saved in the `news/` directory.
 
+### News post caching
+
+News posts on the website are cached for up to sixty days before they are fetched again from GitHub. Purging a news post from the cache is possible, but is limited to a few osu! team members. If you had done a quick-fix on a news post that was already posted/merged, tell [Ephemeral](/users/102335) immediately in the `#osu-wiki` [discord channel](https://discord.gg/ppy) to purge it.
```

Comment:
sixty days? 👀

Reply:
yes, it is set to 86400 minutes (= 60 days)